Tchap/target.yml

```diff
@@ -145,20 +153,33 @@ targets:
     - path: ../Riot/Modules/KeyBackup
     - path: ../Riot/Modules/KeyVerification
     - path: ../Riot/Modules/MediaPicker
+    - path: ../Riot/Modules/PublicRoomList
     - path: ../Riot/Modules/QRCode
     - path: ../Riot/Modules/Reauthentication
     - path: ../Riot/Modules/Room/BubbleReactions
     - path: ../Riot/Modules/Room/DataSources
+    - path: ../Riot/Modules/Room/NotificationSettings
+      excludes:
+        - "RoomNotificationSettingsViewAction.swift"
+        - "RoomNotificationSettingsViewStateType.swift"
+        - "RoomNotificationState.swift"
+    - path: ../Riot/Modules/Room/Views/Avatar
     - path: ../Riot/Modules/Room/Views/BubbleCells/BaseBubbleCell/BubbleCellReactionsDisplayable.swift
     - path: ../Riot/Modules/Room/Views/BubbleCells/RoomTypingBubbleCell.swift
     - path: ../Riot/Modules/Room/Views/CircleProgressView/CircleProgressView.swift
     - path: ../Riot/Modules/Room/Views/URLPreviews
     - path: ../Riot/Modules/RoomKeyRequest/RoomKeyRequestViewController.h
     - path: ../Riot/Modules/RoomKeyRequest/RoomKeyRequestViewController.m
+    - path: ../Riot/Modules/Rooms
+      excludes:
+          - "RoomsViewController.m"
+          - "RoomsViewController.h"
     - path: ../Riot/Modules/Secrets
     - path: ../Riot/Modules/SecureBackup
     - path: ../Riot/Modules/ServiceTerms
     - path: ../Riot/Modules/SlidingModal
+    - path: ../Riot/Modules/Spaces/SpaceRoomList
```

Comment:
Why do you add this resource?

Reply:
This resource is not needed for this PR. I removed it in last commit.